server/chat.js

```diff
@@ -286,7 +286,17 @@ Chat.statusfilters = [];
  * @param {User} user
  */
 Chat.statusfilter = function (status, user) {
+	// Protocol character
 	status = status.replace(/\|/g, '');
+	if (!Config.disablebasicnamefilter) {
+		// e-mail address
+		if (status.includes('@') && status.includes('.')) return '';
```

Comment:
Why not?

Reply:
people have asked for it and i dont really see the legitimate reason for including emails in your status